lib/events/athena/athena.go

```diff
@@ -415,6 +426,17 @@ func (cfg *Config) SetFromURL(url *url.URL) error {
 		}
 		cfg.BatchMaxInterval = dur
 	}
+	consumerLockName := url.Query().Get("consumerLockName")
+	if consumerLockName != "" {
+		cfg.ConsumerLockName = fmt.Sprintf("athena_%s", consumerLockName)
```

Comment:
```suggestion
		cfg.ConsumerLockName = "athena_" + consumerLockName
```

Reply:
Alternatively, given the potential for disallowed characters (such as `/`):
```suggestion
		cfg.ConsumerLockName = fmt.Sprintf("athena_%x", consumerLockName)
```

Reply:
Ah, nice. Updated per suggestion, PTAL.